apps/elewa-website/src/app/app.module.ts

```diff
@@ -6,9 +6,14 @@ import { AppComponent } from './app.component';
 import { NxWelcomeComponent } from './nx-welcome.component';
 import { TranslocoRootModule } from './transloco-root.module';
 
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
 @NgModule({
   declarations: [AppComponent, NxWelcomeComponent],
-  imports: [BrowserModule],
+  imports: [
+    BrowserModule, 
+    ButtonsModule   /*configure button module*/ 
+
+  ],
```

Comment:
No need to push this as the button will be imported by components/modules that need it not the app module